toolchains/android_sdk/BUILD

```diff
@@ -18,7 +18,6 @@ toolchain(
     name = "android_sdk_tools",
     exec_compatible_with = [
         "@platforms//cpu:x86_64",
-        "@platforms//os:linux",
```

Comment:
Yea, we're using this on Mac x86, Mac M1 and Mac M2.

Reply:
Yeah this works for us

Reply:
Cool! TBH I am a little surprised that this actually just works, since the various SDK binaries differ between the Linux and Mac versions, but maybe the toolchain selection for the individual tools just occurs somewhere else.